backend/src/main/java/com/redhat/cloud/notifications/health/KafkaConsumedTotalChecker.java

```diff
@@ -42,25 +39,19 @@ void postConstruct() {
         }
     }
 
-    @Scheduled(every = "${notifications.kafka-consumed-total-checker.period:1m}", delayed = "${notifications.kafka-consumed-total-checker.initial-delay:5m}")
+    @Scheduled(every = "${notifications.kafka-consumed-total-checker.period:5m}", delayed = "${notifications.kafka-consumed-total-checker.initial-delay:5m}")
     public void periodicCheck() {
         if (enabled) {
             double currentTotal = consumedTotalCounter.count();
-            if (currentTotal > previousTotal) {
-                if (currentFailures > 0) {
-                    LOGGER.debugf("Kafka records consumed total check succeeded for topic '%s' after %d failures",
-                            INGRESS_TOPIC, currentFailures);
-                    currentFailures = 0;
-                }
-            } else {
-                currentFailures++;
+            if (currentTotal == previousTotal) {
                 LOGGER.debugf("Kafka records consumed total check failed for topic '%s'", INGRESS_TOPIC);
+                down = true;
             }
```

Comment:
I suppose the health checks would kill the pod before the next execution of this one? (i.e. having a `else { down = false }` would be superfluos?)

Reply:
Yes that's the idea. If `down` is true even once, the pod should be killed.